src/components/ha-selector/ha-selector-select.ts

```diff
@@ -38,6 +42,64 @@ export class HaSelectSelector extends LitElement {
 
   @query("ha-combo-box", true) private comboBox!: HaComboBox;
 
+  private _sortable?: SortableInstance;
+
+  protected updated(changedProps: PropertyValues): void {
+    if (changedProps.has("value")) {
+      if (this.value?.length && !this._sortable) {
+        this._createSortable();
+      } else if (!this.value?.length && this._sortable) {
+        this._destroySortable();
+      }
+    }
+  }
+
+  private async _createSortable() {
+    const Sortable = await loadSortable();
+    this._sortable = new Sortable(
+      this.shadowRoot!.querySelector("ha-chip-set")!,
```

Comment:
We should probably also add sort support to the checkbox option then?

Reply:
I think we should force the dropdown layout if reorder selector option is true. We already do that if there is a custom value. Reordering checkbox is weird.

Reply:
works for me